### DISCUSSION TITLE
Slug validation (formula)

### DISCUSSION THREAD

**Discussion by JoeMatkin** [swellstores/community#291] [Help] [answered]
Hello, what is the exact validation or formula you use to parse and validate slugs. I need to know this is as I'm using a headless CMS and syncing data with Swell and I want to ensure that both slug fields are using the same formula to ensure they are identical.

Thanks

**Comment by logeshswell** [comment, marked as answer]
As mentioned in the [API docs](https://developers.swell.is/backend-api/products/the-product-model), slugs should be,

> Lowercase, hyphenated identifier typically used in URLs with a maximum length of 1,000 characters. When creating a product, a slug will be generated automatically from the name.
